cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs

```diff
@@ -2438,11 +2478,50 @@ instance ToJSON addr
                ]
 
 instance FromJSON PeerSharing where
-  parseJSON = Aeson.withText "PeerSharing" $ \t ->
-    case Text.readMaybe (Text.unpack t) of
-      Nothing -> fail ("PeerSharing.parseJSON: could not parse value: "
-                     ++ Text.unpack t)
-      Just ps -> return ps
+  parseJSON = Aeson.withBool "PeerSharing" $ \b ->
+    pure $ if b then PeerSharingEnabled
+                else PeerSharingDisabled
 
 instance ToJSON PeerSharing where
-  toJSON = String . Text.pack . show
+  toJSON PeerSharingEnabled = Bool True
+  toJSON PeerSharingDisabled = Bool False
+
+instance FromJSON UseLedgerPeers where
+  parseJSON (Number slot) = return $
+    case compare slot 0 of
+      GT -> UseLedgerPeers (After (SlotNo (floor slot)))
+      EQ -> UseLedgerPeers Always
+      LT -> DontUseLedgerPeers
+  parseJSON invalid = fail $ "Parsing of slot number failed due to type mismatch. "
+                            <> "Encountered: " <> show invalid
+
+instance ToJSON LedgerStateJudgement where
+  toJSON YoungEnough = String "YoungEnough"
+  toJSON TooOld      = String "TooOld"
+
+instance FromJSON LedgerStateJudgement where
+  parseJSON (String "YoungEnough") = pure YoungEnough
+  parseJSON (String "TooOld")      = pure TooOld
+  parseJSON _                      = fail "Invalid JSON for LSJ"
+
+instance ToJSON UseLedgerPeers where
+  toJSON DontUseLedgerPeers                  = Number (-1)
+  toJSON (UseLedgerPeers Always)             = Number 0
+  toJSON (UseLedgerPeers (After (SlotNo s))) = Number (fromIntegral s)
+
+instance ToJSON UseBootstrapPeers where
+    toJSON DontUseBootstrapPeers   = Null
+    toJSON (UseBootstrapPeers dps) = toJSON dps
+
+instance FromJSON UseBootstrapPeers where
+    parseJSON Null = pure DontUseBootstrapPeers
+    parseJSON v    = UseBootstrapPeers <$> parseJSON v
```

Comment:
indentation

Reply:
where exactly?

Reply:
The file is indenting with 2 spaces not 4.